Add instructions on how to change Leadfeeder's GTM template (#8)
Leadfeeder/leadfeeder-google-tag-manager-template: README.md

# Leadfeeder Google Tag Manager (GTM) Template

Template for [Leadfeeder](https://www.leadfeeder.com) tag in
[Google Tag Manager](https://tagmanager.google.com)

## How to change the Leadfeeder GTM template

> 1. Motify the template code and add unit tests

Reviewer: ```suggestion
1. Modify the template code and add unit tests.
```

> 1. Import `template.tpl` in your own GTM account

Reviewer: ```suggestion
1. Import `template.tpl` into the existing GTM account.
```

Author: I don't know if this part is clear enough. By "existing account" do we mean Leadfeeder account or developer account?
I created a new GTM account in order to write and test the changes, as I do not have access to the Leadfeeder account.

Reviewer: > into some existing GTM account where it could be tested

?

Author: Sounds good. Done

> 1. Ensure that all unit tests run successfully

Reviewer: ```suggestion
1. Make sure all unit tests run successfully.
```

1. Commit the changes

> 1. Send a PR to review

Reviewer: ```suggestion
1. Create a PR and request a review.
```

1. After the PR is approved, add a change with the last commit SHA to

>    `metadata.yaml`

Reviewer: ```suggestion
1. After a PR has been approved, add the last commit's SHA with a description of the change to the `metadata.yaml` file.
```

> 1. Merge or ask an owner to merge your change (do NOT use squash).
>    * The change will be automatically imported by GTM using the last commit SHA.
>      For more details see https://developers.google.com/tag-manager/templates/gallery

Reviewer: ```suggestion
1. Merge or ask the owner to merge your changes (do NOT use squash). The updated template will be automatically imported by the GTM using that last commit's SHA. For more details see https://developers.google.com/tag-manager/templates/gallery.
```
